docs/core/linux-prerequisites.md

```diff
@@ -17,6 +17,33 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 ## Supported Linux versions
 
+# [.NET Core 3.0 Preview 1](#tab/netcore30p1)
```

Comment:
I believe the name I submitted for the tab name is netcore30

Also, it concerns me a bit that the Preview 1 would be the default tab here with this change. Shouldn't we show 2.0 first?

Reply:
Yes... I'll put it AFTER 1.x since it's a preview?

Reply:
I think that's ok. It's kind of unfortunate we can't set each one is the default and have a proper order.